tests/test_config.py

```diff
@@ -35,3 +41,28 @@ def test_configure_sentry_env_variable_is_dsn(monkeypatch):
     monkeypatch.setenv("SENTRY_DSN", "https://1234567890@00000.ingest.sentry.io/123456")
     result = configure_sentry()
     assert result == "Sentry DSN found, exceptions will be sent to Sentry with env=test"
+
+
+def test_sentry_scope_variables_removed_from_sent_event(
+    mocker,
```

Comment:
[Non-blocking] Where does this fixture come from? I looked for it in `conftest` and by searching for "mocker" in the repo but couldn't find its definition. 🤔

Reply:
Good question!  It looks as though `pytest-mock` adds this as a default fixture to tests when installed.